src/utils/stylize.ts

```diff
@@ -0,0 +1,30 @@
+const colors = {
```

Comment:
Have you heard my rant about "utils" yet :stuck_out_tongue: ?  "friends don't let friends dump things in utils/ or utils.ts"

my not really data-informed opinion: utils/ or utils.ts encourage poorly thought-out implementations and organization. they grow without bound.  so does their usage.  pretty soon everything is being imported from utils.  there's this sprawling mess of unit tests nobody wants to touch. people are extending the poorly thought-out things dumped into utils in poorly thought-out ways.

my not really data-informed solution: if I avoid putting stuff in "utils" it forces me to think a little bit more about the implementation and avoids the anarchy I describe above.

So anyways, nothing is really wrong with the code in stylize.ts, except that it encourages dumping into utils by example.  Consider moving it to Logger.ts (which I realize is also in utils, but maybe it shouldn't be either).

Reply:
Ahah yes, I had it in Logger initially, but it didn't feel right either.
For me, utils serves as a "staging" zone. Code there may upgrade to a dedicated package in the future if it becomes generally useful to more than this project.
Usually it needs to be somewhere so it can be used from more than 1 place. I don't really have a better name than `utils` for this now.